lib/thrift/protocols/binary.ex

```diff
@@ -53,10 +49,9 @@ defmodule Thrift.Protocols.Binary do
   def serialize(_, nil) do
     []
   end
```

Comment:
That sounds right to me. I'll remove it in its own PR so we can track the specific motivation.